interface-definitions/interfaces-wwan.xml.in

```diff
@@ -36,6 +36,67 @@
           <leafNode name="mtu">
             <defaultValue>1430</defaultValue>
           </leafNode>
+          <node name="ping-check">
+            <properties>
+              <help>Use ping check rather than interface state check for automatic re-dial</help>
+            </properties>
+            <children>
+              <leafNode name="enable">
+                <properties>
+                  <help>Enable ping check instead of interface state check</help>
+                  <valueless/>
+                </properties>
+              </leafNode>
+              <leafNode name="address">
+                <properties>
+                  <help>Ping target</help>
+                  <constraint>
+                    <validator name="ipv4-address"/>
+                    <validator name="ipv6-address"/>
+                    <validator name="fqdn"/>
+                  </constraint>
+                  <valueHelp>
+                    <format>ipv4</format>
+                    <description>IP address of ping target</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>ipv6</format>
+                    <description>IPv6 address of ping target</description>
+                  </valueHelp>
+                  <valueHelp>
+                    <format>hostname</format>
+                    <description>Fully qualified domain name of ping target</description>
+                  </valueHelp>
+                </properties>
+              </leafNode>
+              <leafNode name="count">
+                <properties>
+                  <help>Number of requests to send</help>
+                  <valueHelp>
+                    <format>u32:1-655340</format>
+                    <description>Amount</description>
+                  </valueHelp>
+                  <constraint>
+                    <validator name="numeric" argument="--range 1-655340"/>
+                  </constraint>
+                </properties>
+                <defaultValue>4</defaultValue>
+              </leafNode>
+              <leafNode name="deadline">
```

Comment:
I think timeout is a better and more common term

Reply:
I think so too, but I found this term inside [ping command](https://github.com/vyos/vyos-1x/blob/8cf1a827e09f57efd5a4c38226048d15ebbf4a21/src/op_mode/ping.py#L67), so, for this one, should I use timeout or deadline?

Reply:
If you wan‘t a inob please use disable instead

Reply:
Please go with timeout